provisioning_templates/provision/kickstart_default.erb

```diff
@@ -40,7 +40,11 @@ This template accepts the following parameters:
   section_end = (rhel_compatible && os_major <= 5) ? '' : '%end'
 %>
 install
+<% if host_param('kickstart_liveimg') %>
+liveimg --url=<%= host_param('kickstart_liveimg') %> <% unless proxy_string.empty? %> --proxy=<%= proxy_string %><% end %>
```

Comment:
Please use the same `proxy_string` logic as two lines below (always display it, also when it's empty), see line 35 & 36 for the logic behind it.

Reply:
right, sorry, changed it.